packages/tenant-process/test/testAddCertifiedAttribute.ts

```diff
@@ -39,56 +39,61 @@ import {
 
 export const testAddCertifiedAttribute = (): ReturnType<typeof describe> =>
   describe("addCertifiedAttribute", async () => {
+    let requesterTenant: Tenant;
+    let attribute: Attribute;
+
     const tenantAttributeSeed: ApiCertifiedTenantAttributeSeed = {
       id: generateId(),
     };
-    const correlationId = generateId();
-    const requesterTenant: Tenant = {
-      ...getMockTenant(),
-      features: [
-        {
-          type: "PersistentCertifier",
-          certifierId: generateId(),
-        },
-      ],
-      updatedAt: currentDate,
-    };
-    const attribute: Attribute = {
-      name: "an Attribute",
-      id: unsafeBrandId(tenantAttributeSeed.id),
-      kind: attributeKind.certified,
-      description: "an attribute",
-      creationTime: new Date(),
-      code: "123456",
-      origin: requesterTenant.features[0].certifierId,
-    };
-
     const targetTenant: Tenant = getMockTenant();
-    const organizationId = getMockAuthData(requesterTenant.id).organizationId;
 
-    it("Should add the certified attribute if the tenant doesn't have that", async () => {
+    beforeAll(async () => {
       vi.useFakeTimers();
       vi.setSystemTime(new Date());
+
+      requesterTenant = {
+        ...getMockTenant(),
+        features: [
+          {
+            type: "PersistentCertifier",
+            certifierId: generateId(),
+          },
+        ],
+        updatedAt: new Date(),
+      };
+
+      attribute = {
+        ...getMockAttribute(),
+        id: unsafeBrandId(tenantAttributeSeed.id),
+        kind: attributeKind.certified,
+        origin: requesterTenant.features[0].certifierId,
+      };
+    });
+
+    afterAll(() => {
+      vi.useRealTimers();
+    });
+
+    it("Should add the certified attribute if the tenant doesn't have that", async () => {
       await addOneAttribute(attribute, attributes);
       await addOneTenant(targetTenant, postgresDB, tenants);
       await addOneTenant(requesterTenant, postgresDB, tenants);
       await tenantService.addCertifiedAttribute(targetTenant.id, {
         tenantAttributeSeed,
-        organizationId,
-        correlationId,
+        organizationId: getMockAuthData(requesterTenant.id).organizationId,
```

Comment:
```suggestion
        organizationId: generateId(),
```

Reply:
If I use `generateId()` instead of `getMockAuthData(requesterTenant.id).organizationId` I get the `tenantNotFound` error so you have to pass it the organizationId

Reply:
> If I use `generateId()` instead of `getMockAuthData(requesterTenant.id).organizationId` I get the `tenantNotFound` error so you have to pass it the organizationId

You're right, it should just be `requesterTenant.id`